hawkular-alerts-api/src/main/java/org/hawkular/alerts/api/model/event/EventCategory.java

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright 2015 Red Hat, Inc. and/or its affiliates
+ * and other contributors as indicated by the @author tags.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.hawkular.alerts.api.model.event;
+
+/**
+ * Every Event has a category to help identify what it represents.  It can be one of these, or it can be something
+ * specified by the Event creator.  This just tries to predefine common events.
+ *
+ * @author jay shaughnessy
+ * @author lucas ponce
+ */
+public enum EventCategory {
+    ALERT, DEPLOYMENT, LOG, TRIGGER
```

Comment:
Good eyes :)  That is what it means but honestly it is pretty weak because TRIGGER does not really explain what the event is truly about, only the source.  But the alternative would be to add yet another field to Trigger, used only on Event triggers, to specify a category.  I suppose one possibility would be to take the current Trigger.eventText field and instead make it Trigger.category.  And then, set the Event text explicitly to the Trigger name.  That may not be bad at all. Let me know...